### PR TITLE
Replaced "master" with newer terms in two places

### PR DESCRIPTION
**Pull request by JayBeale**
- Replacing the outdated "master" term with API server.
- Changed "master" to "control plane node(s)"

### CODE REVIEW

File: content/en/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -335,7 +335,7 @@ the same time:
   (default 0.01) per second.
 
 The reason these policies are implemented per availability zone is because one
-availability zone might become partitioned from the master while the others remain
+availability zone might become partitioned from the control plane node(s) while the others remain
```
Comment (sftim) [inline content/en/docs/concepts/architecture/nodes.md:338]:
```suggestion
availability zone might become partitioned from the control plane while the others remain
```

You might well run your control plane on nodes, but it's not obligatory.

**Review by sftim — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/architecture/nodes.md (hunk 1 above)
Comment (reylejano) [reply content/en/docs/concepts/architecture/nodes.md:338]: +1 to @sftim 's suggestion

Comment (JayBeale) [reply content/en/docs/concepts/architecture/nodes.md:338]: Nice! Thanks @sftim and @reylejano !

**Pull request merged**